lib/tasks/test_unit.rake

```diff
@@ -2,6 +2,8 @@ namespace :test do
   desc "Runs unit tests."
   task "unit" do
     files = Dir["**/test/**/*_{spec,test}.rb"]
+    # sort the files to have reproducible runs
+    files.sort!
```

Comment:
well order depend tests are pain and I am not sure if we should workaround it. We already print order for which it failed, which is good for me. So I have still same opinion :)